.gitignore

```diff
@@ -6,3 +6,4 @@ tmp
 logs
 .DS_Store
 test-results.xml
+docs/
```

Comment:
why do you ignore the `docs` ? I think they should be part of the current released repository.